packages/kbn-cli-dev-mode/src/dev_server.ts

```diff
@@ -215,8 +215,8 @@ export class DevServer {
         // handle graceful shutdown requests
         const triggerGracefulShutdown$ = gracefulShutdown$.pipe(
           mergeMap(() => {
-            // signal to the process that it should exit
-            proc.kill('SIGINT');
+            // no need to signal to the process that it should exit. execa already does that for us.
+            // proc.kill('SIGINT');
```

Comment:
I think I'd prefer that we set `cleanup: false` when we spawn the server rather than relying on execa for this

Reply:
I'm wrong... the `cleanup` option must be something else. No matter the value `true/false`, the `SIGINT` event still reaches the child process. I'll keep testing on Monday to figure out how to control it on our end instead of relying on some dark magic in Node that is spreading the signals to both parent and child.

What I'm observing is me triggering the signal `SIGINT`, and `dev_server` replicating it. So the Kibana server receives 2 signals in a row and shuts down.

Reply:
@spalger I've reverted the changes and created this other issue as a follow up: #97439. Since it affects the dev mode only, I don't think it should delay the production graceful shutdown. What do you think?